community/self-hosting-novu/deploy-with-docker.mdx

```diff
@@ -158,3 +158,32 @@ This would producte the following url: - API: company.com/novu-api
   These env variables should be present on all services novu provides due to tight
   coupling.
 </Note>
+
+Here’s an improved version of your write-up with additional details and clarification:
```

Comment:
Please remove

Reply:
Should we maybe call this section `FAQ`? I feel it needs a title.

Reply:
Removed. I was also confused with title. I did not keep it FAQs because I only talked about Local Tunnel and Self-Hosted Deployments. Let me rename and reorganise